test/test_train_imagenet.py

```diff
@@ -180,12 +203,16 @@ def test_loop_fn(model, loader, device, context):
 
   accuracy = 0.0
   writer = SummaryWriter(log_dir=FLAGS.logdir) if FLAGS.logdir else None
+  num_training_steps_per_epoch = len(train_dataset.imgs) // (
+      FLAGS.batch_size * (len(devices) or 1))
   for epoch in range(1, FLAGS.num_epochs + 1):
     model_parallel(train_loop_fn, train_loader)
     accuracies = model_parallel(test_loop_fn, test_loader)
     accuracy = mean(accuracies)
     print("Epoch: {}, Mean Accuracy: {:.2f}%".format(epoch, accuracy))
-    test_utils.add_scalar_to_summary(writer, 'Accuracy/test', accuracy, epoch)
+    global_step = (epoch - 1) * num_training_steps_per_epoch
```

Comment:
is -1 intended here?

Reply:
Yes - I want the x-axis of the learning rate and the accuracy graphs to match. This accuracy graph will start at step 0, then log another point at num_training_steps_per_epoch, etc.